internal/internal_worker.go

```diff
@@ -783,28 +795,34 @@ func (aw *aggregatedWorker) Start() error {
 		return fmt.Errorf("failed to get executable checksum: %v", err)
 	}
 
+	workerStarted := false
+
 	if !isInterfaceNil(aw.workflowWorker) {
 		if len(aw.registry.getRegisteredWorkflowTypes()) == 0 {
-			aw.logger.Warn(
-				"Starting worker without any workflows. Workflows must be registered before start.",
+			aw.logger.Info(
+				"Worker has no workflows registered. Workflows must be registered before start. Skipping",
```

Comment:
"Worker has no workflows registered. Workflows must be registered before start. Skipping" -> "Worker has no workflows registered, so workflow worker will not be started."
Same for activity worker.

Reply:
thanks!